src/assets/i18n/en.json5

```diff
@@ -1571,6 +1571,112 @@
   "home.top-level-communities.help": "Select a community to browse its collections.",
 
 
+  "handle-table.breadcrumbs": "Handles",
+
+  "handle-table.new-handle.breadcrumbs": "New Handle",
+
+  "handle-table.edit-handle.breadcrumbs": "Edit Handle",
+
+  "handle-table.global-actions.breadcrumbs": "Global Actions",
+
+
+  "handle-table.new-handle.form-handle-input-text": "Handle",
+
+  "handle-table.new-handle.form-handle-input-placeholder": "Enter handle",
+
+  "handle-table.new-handle.form-url-input-text": "URL",
+
+  "handle-table.new-handle.form-url-input-placeholder": "Enter URL",
+
+  "handle-table.new-handle.form-button-submit": "Submit",
+
+
+  "handle-table.new-handle.notify.error": "Server Error - Cannot create a new handle",
```

Comment:
Cannot create new handle. No need for 'a'

Reply:
Removed 'a'